journal/src/main/scala/com/evolutiongaming/kafka/journal/Journals.scala

```diff
@@ -78,7 +78,7 @@ object Journals {
     eventualJournal: EventualJournal[F],
     headCache: HeadCache[F],
     log: Log[F]
-  ): Journals[F] = {
+  )(implicit jsValueCodec: JsValueCodec): Journals[F] = {
```

Comment:
could you please be consistent on naming things?

decide upon one name and not use interchangeably `jsCodec` and `jsValueCodec`